debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (quilt)
```

Comment:
```suggestion
3.0 (native)
```

Since we're not packaging an external source tarball we should use `native` here. Otherwise creating source packages with `dpkg-buildpackage` won't work (only binary-only packages with `dpkg-buildpackage --build=binary`).